src/renderer/components/wallet/txs/SendForm.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useCallback, useMemo } from 'react'
+
+import * as RD from '@devexperts/remote-data-ts'
+import { bn, assetToString, Asset, formatAssetAmountCurrency, assetAmount } from '@thorchain/asgardex-util'
+import { Row, Form } from 'antd'
+import { Store } from 'antd/lib/form/interface'
+import * as FP from 'fp-ts/lib/function'
+import { useIntl } from 'react-intl'
+import { useHistory } from 'react-router-dom'
+
+import * as walletRoutes from '../../../routes/wallet'
+import { SendTxParams } from '../../../services/binance/transaction'
+import { AssetWithBalance, AssetsWithBalanceRD, AssetsWithBalance } from '../../../services/binance/types'
+import { Input, InputNumber } from '../../uielements/input'
+import AccountSelector from './../AccountSelector'
+import * as Styled from './Send.style'
+
+type Props = {
+  assetsWB: AssetsWithBalanceRD
+  assetWB: AssetWithBalance
+  onSubmit: ({ to, amount, asset, memo }: SendTxParams) => void
+  isLoading: boolean
```

Comment:
do we really want to use loading flags ? i mean what is a sense to handle form's state 'isLoading' instead of shohwing it only in case of initial (as it was) and sow `SuccessView` in case of success ?

Reply:
Yep, we do need this flag, since we don't show a "loading screen" anymore, We will still stay at the form while having a "loading" state. The form will show a spin in its submit button and all input fields are disabled.